pkg/watcher/skr_webhook_manifest_manager.go

```diff
@@ -62,6 +65,10 @@ func NewSKRWebhookManifestManager(kcpClient client.Client,
 	if err != nil {
 		return nil, err
 	}
+	kcpClient, err := client.New(kcpConfig, client.Options{Scheme: schema})
+	if err != nil {
+		return nil, err
```

Comment:
Would you mind if you wrap this error as soon the `wrapcheck` will be introduced. Thanks!

Reply:
done

Reply:
Thanks!